### DISCUSSION TITLE
what's the meaning of `K3S_CLUSTER_INIT`

### DISCUSSION THREAD

**Discussion by xiaowu-wang950330** [k3s-io/k3s#5974] [Q&A] [answered]
what's the meaning of `K3S_CLUSTER_INIT`

**Comment by wang-xiaowu** [comment, marked as answer]
it's about High availability that related to the etcd storage

**Comment by brandond** [comment]
Most of the K3s CLI flags (such as --cluster-init) can also be set using an environment variable with the K3S_ prefix. So `K3S_CLUSTER_INIT=true k3s server` is the same as `k3s server --cluster-init=true`.
